doc/admin-guide/files/records.config.en.rst

```diff
@@ -272,6 +272,12 @@ System Variables
 
    Specifies at what size to roll the output log at.
 
+.. ts:cv:: CONFIG proxy.config.output.logfile.rolling_min_count INT 0
+   :reloadable:
+
+   Specifies the minimum count of rolled output logs to keep. This value will be used to decide the
```

Comment:
Minor detail, but do we want to be explicit about what happens if the disks are getting close to being filled up? I assume at that point, it'll ignore these settings, and start to aggressively delete old logs anyways?

Reply:
If so, it should explicitly mention the headroom configuration value, which is what "close to being filled up" means.